sagemaker-inference-recommender/deployment-recommendations/instant-recommendations.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<p>Please clear output to keep notebook clean</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4077/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>ack</p>